docs/source/tutorial.rst

```diff
@@ -176,6 +176,14 @@ If you want to **lower contract value**, you can insert new one into the `amount
 
 `200 OK` response was returned. The value was modified successfully.
 
+Setting contract.item.unit.value
```

Comment:
Setting value per item's unit
~~~~~~~~~~~~~~~~~~~~~~~~~~

``contract.item.unit.value`` can be set only for **reporting** tenders.